containers/Makefile

```diff
@@ -117,18 +117,18 @@ build/concent-api/concent-api.tar: build/repositories/concent/  build/virtualenv
 build/nginx-proxy/concent-api-assets.tar: build/repositories/concent/ build/virtualenv/ build/package-builder/build-concent-api-assets.sh
 	mkdir --parents "$(dir $@)"
 	build/package-builder/build-concent-api-assets.sh\
-		"build/repositories/concent/"                  \
-		"$(abspath package-builder)"                   \
-		"$(abspath build)"                             \
-		"$(dir $@)"
+	  "build/repositories/concent/"                  \
```

Comment:
I'm not sure if having a mix of tabs and spaces at the beginning of a line is a good idea.

Reply:
according to coding style, I used tab where is needed by makefile syntax, and space everywhere else (shell arguments)